alf/algorithms/rl_algorithm.py

```diff
@@ -293,7 +293,23 @@ def observe(self, exp: Experience):
         for observer in self._exp_observers:
             observer(exp)
 
-    def training_summary(self, training_info, loss_info, grads_and_vars):
+    def summarize_rollout(self, training_info):
+        if self._debug_summaries:
+            summary_utils.add_action_summaries(
+                training_info.action, self._action_spec, "rollout_action")
+            self.add_reward_summary("rollout_reward/extrinsic",
+                                    training_info.reward)
+
+        if self._summarize_action_distributions:
+            field = nest_utils.find_field(training_info.info,
```

Comment:
Why do we perform ```summarize_rollout``` using ```training_info.info``` instead of ```training_info.rollout_info```? This part is the same as ```summarize_train```.
Maybe useful to document what are the fields of ```training_info```  are expected to be used in this function (and ```summarize_train```).

Reply:
policy_step.info is saved to training_info.info. I'll change it to rollout_info.